services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/BandwidthSpec.groovy

```diff
@@ -63,4 +67,53 @@ class BandwidthSpec extends BaseSpecification {
         cleanup: "Delete created flow"
         flow?.id && northboundService.deleteFlow(flow.id)
     }
+
+    def "Unable to exceed bandwidth limit on ISL when creating a flow"() {
+        given: "Two active switches"
+        def switches = topology.getActiveSwitches()
+        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
+                .find { Switch src, Switch dst -> src != dst }
+        assert srcSwitch && dstSwitch
+
+        when: "Create a flow with a bandwidth that exceeds available bandwidth on ISL (ignore_bandwidth = false)"
+        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        def possibleFlowPaths = topologyEngineService.getPaths(srcSwitch.dpId, dstSwitch.dpId)*.path
+        def allLinks = northboundService.getAllLinks()
+        def involvedBandwidths = []
+
+        possibleFlowPaths.each { path ->
+            pathHelper.getInvolvedIsls(path).each { link ->
+                involvedBandwidths.add(islUtils.getIslInfo(allLinks, link).availableBandwidth)
+            }
+        }
+        flow.maximumBandwidth = involvedBandwidths.max() + 1
+        flow.ignoreBandwidth = false
+        northboundService.addFlow(flow)
+
+        then: "Flow is not created because flow path should not be found"
+        def exc = thrown(HttpClientErrorException)
+        exc.rawStatusCode == 404
+    }
+
+    def "Able to exceed bandwidth limit on ISL when creating a flow"() {
```

Comment:
This name is not corresponding with the previos test name. You cannot have two tests
`Unable to exceed bandwidth limit on ISL when creating a flow`
`Able to exceed bandwidth limit on ISL when creating a flow`
Not clear what is the expected system behavior. You should specify that this scenario uses ignoreBandwidth as an exceptional case